Week2/homework/map_filter.js

```diff
@@ -3,3 +3,7 @@
 const numbers = [1, 2, 3, 4];
 
 // Add your code here
+let newNumbers = numbers.filter((number) => number % 2 !== 0);
```

Comment:
Hi @Moataz7 - we don't really need to use `let` here to create the `newNumbers` variable. Try to make use of `const` please. Instead of doing the filter & map operations in two different statements, you can try to use method chaining using the `dot(.)` operator and then you can assign the output of it to the `const` newNumbers.

I would also recommend that you try to use `const` as much as possible. Since it is a non-changing(almost a constant) variable, we can be assured that the value we assign to it is going to remain the same throughout. If I use `let` here, in the future, you or another developer can accidentally change its value which can then potentially cause bugs in our app.

Also, if we are passing a single argument using arrow function, we don't need `paranthesis()`. You can have `filter(number => number % 2 !== 0);` instead. Same goes for the map operation below. 🙂